client/dashboard/task-list/index.js

```diff
@@ -256,6 +256,58 @@ class TaskDashboard extends Component {
 		);
 	}
 
+	onSkipStoreSetup = () => {
+		const completedTasksNames = this.getTasks()
+			.filter( x => x.completed )
+			.map( x => x.title );
+
+		recordEvent( 'wcadmin_tasklist_skip', {
+			completed_tasks: completedTasksNames.length,
+			completed_tasks_names: completedTasksNames,
+			skip_option: 'skip store setup'
+		} );
+
+		this.props.updateOptions( {
+			woocommerce_task_list_skip_store_setup: true,
+		} );
+	};
+
+	onDoThisLater = () => {
+		const completedTasksNames = this.getTasks()
+			.filter( x => x.completed )
+			.map( x => x.title );
+
+		recordEvent( 'wcadmin_tasklist_skip', {
+			completed_tasks: completedTasksNames.length,
+			completed_tasks_names: completedTasksNames,
+			skip_option: 'i\'ll do this later'
+		} );
+
+		this.props.updateOptions( {
+			woocommerce_task_list_do_this_later: true,
+		} );
+	};
+
+	renderSkipActions() {
+		const { skipStoreSetup, doThisLater } = this.props;
+
+		return (
+			<div style={ { textAlign: 'center' } }>
+				{ ! skipStoreSetup && (
```

Comment:
I think we can simplify this logic as the task list won't be shown at all if they skip, right?